data_steward/utils/bq.py

```diff
@@ -93,7 +117,7 @@ def create_tables(client, project_id, fq_table_names, exists_ok=False):
     successes = []
     failures = []
     for table_name in fq_table_names:
-        schema = get_table_schema(table_name.split('.')[2])
+        schema = get_table_schema(table_name.split('.')[2], fields)
```

Comment:
The changes here are great!  My only concern is if someone sends a list of table names and a single table fields/schema definition to `create_tables`.  I think `fields` may need to be a list of schemas sent to `create_tables`.  Then the `for` loop could be
```
for index, table_name in enumerate(fq_table_names):
    schema = get_table_schema(table_name.split('.')[2], fields[index] if fields else None)
```
FYI:  I haven't tested this.

Reply:
Ahh I see what you're saying. I've made these adjustments and made my list a list of lists of JSON objects. I tested it with my integration test and it will throw an error saying index out of range, if I supply two tables and only one list for the fields/schema.